src/App.jsx

```diff
@@ -145,8 +143,8 @@ function App() {
                 <Switch>
                     <ProtectedRouteWithProps exact path='/auth' component={AuthCallback} />
                     <ProtectedRouteWithProps exact path="/" component={Home} />
-                    <ProtectedRouteWithProps path="/sites/:siteName/collections/:collectionName/:fileName" component={EditPage} isCollectionPage={true} isResourcePage={false} />
-                    <ProtectedRouteWithProps path="/sites/:siteName/collections/:collectionName" component={CategoryPages} isResource={false}/>
+                    <ProtectedRouteWithProps exact path="/sites/:siteName/folder/:collectionName/subfolder/:subfolderName/:fileName" component={EditPage} isCollectionPage={true} isResourcePage={false} />
+                    <ProtectedRouteWithProps exact path="/sites/:siteName/folder/:collectionName/:fileName" component={EditPage} isCollectionPage={true} isResourcePage={false} />
```

Comment:
Nit: Can we standardize as`:folderName` as with the two lines below?

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/379/commits/c293096ad096319f0fc2d50e3fe2d8f943d7fa78